app/models/topic.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
```

Comment:
You should run the migrations again so that the annotate comments include the index here